cmd/flinkk8soperator/cmd/root.go

```diff
@@ -75,6 +76,16 @@ func init() {
 	// allows `$ flinkoperator --logtostderr` to work
 	klog.InitFlags(nil)
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+
+	logrus.SetFormatter(&logrus.JSONFormatter{
```

Comment:
It would be nice to move it into a package (please see https://github.com/sirupsen/logrus#formatters) and introduce a flag to set appropriate formatter.

Reply:
Could you provide me more information how to achieve that? or links with examples how to do that.